src/predict_api/flask_app.py

```diff
@@ -0,0 +1,243 @@
+import json
+
+import requests
+from apispec import APISpec
+from apispec_webframeworks.flask import FlaskPlugin
+from apispec.ext.marshmallow import MarshmallowPlugin
+from dotenv import load_dotenv
+from flask import Flask, g, redirect, request, url_for, jsonify
+from flask_login import (
+    LoginManager,
+    login_required,
+    login_user,
+    logout_user,
+)
+from flask_cors import CORS
+from flask_sqlalchemy import SQLAlchemy
+from oauthlib.oauth2 import WebApplicationClient
+
+from config import Config, logging
+
+from flask import jsonify, render_template, Blueprint
+from flask_login import login_required
+
+from config import logging, Config
+
+
+
+# TODO: many below
+# figure out topology. This should run in a separate container. Are requests all proxied through the annotation api?
+# auth - either it runs behind the main container or we need separate auth
```

Comment:
The annotation API is currently using a session-based auth, not token-based.
It seems inevitable that we will want token-based auth when we have both react app and native/mobile apps using the same API, but that seems like an optimisation that can be delayed until we need it (YAGNI principle).
So for the time being it seems like maybe we should be proxying this behind the annotation API to leverage the session-auth, but we could stop doing that when we move to token-based auth on both the annotation API and the prediction API.

Reply:
I'll defer to you and your folks on the best way to design this.

Reply:
why not merge these APIs? /api/v0/annotation/ vs /api/v0/prediction/
It removes all the duplicated boilerplate and we just have separate python files for different endpoints (like `predictions.py` file here)
It mixes things up - annotation endpoints could access prediction machinery configured because they have the same env variables - but in the end decreases complexity

Reply:
that would postpone/avoid the nee for token-based auth... see, YAGNI :)